resources/make_test_resources.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+# This script illustrates the build steps for all the kernel and disk images
+# used by tests in vmm-reference.
+
+set -e
+
+SOURCE=$(readlink -f "$0")
+TEST_RESOURCE_DIR="$(dirname "$SOURCE")"
+cd $TEST_RESOURCE_DIR
+
+./kernel/make_kernel_busybox_image.sh -f elf -k vmlinux-hello-busybox -w /tmp/vmlinux_busybox -j 2
```

Comment:
nit: you could set variables with the various locations in `tmp` so that they're easier to modify; you could also check if the variable is already set and only modify it if it's unset (so that users who want different locations for the artifacts can do something like `KERN_DIR=/home/user/kernel ./make_test_resources.sh`)

Reply:
done